sdk/ml/azure-ai-ml/azure/ai/ml/_schema/registry/registry.py

```diff
@@ -30,7 +30,7 @@ class RegistrySchema(ResourceSchema):
         casing_transform=snake_to_pascal,
     )
     replication_locations = fields.List(NestedField(RegistryRegionDetailsSchema))
-    intellectual_property_publisher = fields.Str()
+    intellectual_property = fields.Str()
```

Comment:
Do we need to change this? - from my understanding it should either be IntellectualProperty  object or we should keep it as is (intellectual_property_publisher string). cc @vikasagrawal-ms

Reply:
I have updated this to be an object to match the way we should define the YAMLs from the spec.